project/build.properties

```diff
@@ -1,4 +1 @@
-#Activator-generated Properties
-#Sat Jan 09 19:24:14 CET 2016
-template.uuid=c331da07-ace3-45a5-bc6d-bf11e5d6385c
-sbt.version=0.13.8
+sbt.version=0.13.9
```

Comment:
Oops. I updated the sbt version. Was not the intention - are you okay with that?

The template.uuid is placed by Activator. It's not really needed, as far as I know.